### ISSUE TITLE
PySide 5.15.2 causes linux tests to fail on CI

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#1920]
## 🐛 Bug

Most of our linux tests are failing right now, and i'm not sure why. I noticed it a couple days ago :-(

I'm hoping it's just a CI think that will resolve, but I'm making this issue so we can have discussion

https://github.com/napari/napari/runs/1433860671

<details>

```
  File "/root/conda/lib/python3.7/threading.py", line 890 in _bootstrap

Current thread 0x00007f795adc1740 (most recent call first):
  File "/root/conda/lib/python3.7/site-packages/pytestqt/plugin.py", line 57 in qapp
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 930 in call_fixture_func
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 1124 in pytest_fixture_setup
  File "/root/conda/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/root/conda/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 1070 in execute
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 689 in _compute_fixture_value
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 605 in _get_active_fixturedef
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 1046 in execute
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 689 in _compute_fixture_value
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 605 in _get_active_fixturedef
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 1046 in execute
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 689 in _compute_fixture_value
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 605 in _get_active_fixturedef
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 585 in getfixturevalue
  File "/root/conda/lib/python3.7/site-packages/_pytest/fixtures.py", line 572 in _fillfixtures
  File "/root/conda/lib/python3.7/site-packages/_pytest/python.py", line 1633 in setup
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 448 in prepare
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 151 in pytest_runtest_setup
  File "/root/conda/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/root/conda/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 256 in <lambda>
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 310 in from_call
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 256 in call_runtest_hook
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 216 in call_and_report
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 121 in runtestprotocol
  File "/root/conda/lib/python3.7/site-packages/_pytest/runner.py", line 110 in pytest_runtest_protocol
  File "/root/conda/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/root/conda/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/root/conda/lib/python3.7/site-packages/_pytest/main.py", line 338 in pytest_runtestloop
  File "/root/conda/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/root/conda/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/root/conda/lib/python3.7/site-packages/_pytest/main.py", line 313 in _main
  File "/root/conda/lib/python3.7/site-packages/_pytest/main.py", line 257 in wrap_session
  File "/root/conda/lib/python3.7/site-packages/_pytest/main.py", line 306 in pytest_cmdline_main
  File "/root/conda/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/root/conda/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/root/conda/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/root/conda/lib/python3.7/site-packages/_pytest/config/__init__.py", line 165 in main
  File "/root/conda/lib/python3.7/site-packages/_pytest/config/__init__.py", line 187 in console_main
  File "/root/conda/bin/pytest", line 8 in <module>
/tmp/scripts4592d2572bcd0668d2d6c52f5054e2d0.sh: line 8:  3568 Aborted                 (core dumped) pytest -v --cov-report=xml --cov=./napari

Exit status: 134
```

</details>

**Comment by sofroniewn** [comment]
I'll add that versions installed look the same for both a [passing test](https://cirrus-ci.com/task/5858090258006016) from two days ago and a [failing test](https://cirrus-ci.com/task/6389165078085632?command=install#L108) from yesterday, so it doesn't look like there was a new package released, but more likely something on the CI side

**Comment by Czaki** [comment]
> I'll add that versions installed look the same for both a [passing test](https://cirrus-ci.com/task/5858090258006016) from two days ago and a [failing test](https://cirrus-ci.com/task/6389165078085632?command=install#L108) from yesterday, so it doesn't look like there was a new package released, but more likely something on the CI side

The main important difference is PySide2 version. Two days ago it was 5.15.1 and not it is 5.15.2. I think it should be checked first.

**Comment by Czaki** [comment]
@sofroniewn please see # #1925

**Comment by sofroniewn** [comment]
> The main important difference is PySide2 version. Two days ago it was 5.15.1 and not it is 5.15.2. I think it should be checked first.

Wow, i completely missed that!! Thanks

**Comment by sofroniewn** [comment]
So we know the issue is `5.15.2` and that tests can pass locally. I wonder if the following has any useful information? https://wiki.qt.io/Qt_5.15.2_Known_Issues

**Comment by Czaki** [comment]
There is also PyQt5 released (2 minutes ago). It should be checked.

**Comment by Czaki** [comment]
I found proper bug report https://bugreports.qt.io/browse/QTBUG-88688

**Comment by Czaki** [comment]
So the option is to switch from gcc:latest to some ubuntu based image.

**Comment by Czaki** [comment]
So I have little of time today and propose some fix in  #1937

**Comment by Czaki** [comment]
@sofroniewn close?

**Comment by sofroniewn** [comment]
Yes, thanks!! Closed by #1937
